op-node/rollup/superchain.go

```diff
@@ -59,13 +60,15 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 	}
 
 	regolithTime := uint64(0)
-	// two goerli testnets test-ran Bedrock and later upgraded to Regolith.
+	// three goerli testnets test-ran Bedrock and later upgraded to Regolith.
 	// All other OP-Stack chains have Regolith enabled from the start.
 	switch chainID {
 	case baseGoerli:
 		regolithTime = 1683219600
 	case opGoerli:
 		regolithTime = 1679079600
+	case labsDevnet:
+		regolithTime = 1677984480
 	}
```

Comment:
Similar to @trianglesphere's comment, is there a way to store the regolith timestamp in a per-chain config rather than putting it into the config loader function? What's the idea behind putting some chain-specific configuration into a seemingly generic loader function?

Reply:
The idea was that goerli and devnet will stop to exist, and so I thought it would be better to not bloat the registry with configuration options that nobody should change. Same thing for the other parameters like channel-timeout, which should be the same with every network.

I can add it as option to the registry, to reduce the complexity here in the op-node, but it does the opposite in the registry, adding more complexity there. It's a trade-off.

Reply:
I see the argument that per-chain hf activation times should be out of scope for the superchain-registry. It's only a temporary situation and it can be deleted once Goerli shuts down. So I'm happy to keep as is.